website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -979,6 +981,8 @@ The `oms_agent` block exports the following:
 
 * `oms_agent_identity` - An `oms_agent_identity` block is exported. The exported attributes are defined below.
 
+* `msi_auth_for_monitoring_enabled` - Is managed identity authentication for monitoring enabled?
+
```

Comment:
Only properties that the user can't set and are determined by the provider are added to this part of the documentation where we detail exported attributes.
```suggestion
* `oms_agent_identity` - An `oms_agent_identity` block is exported. The exported attributes are defined below.

```